lib/web/apiserver.go

```diff
@@ -563,6 +565,52 @@ func (h *Handler) getUserStatus(w http.ResponseWriter, r *http.Request, _ httpro
 	return OK(), nil
 }
 
+// handleGetUserOrResetToken has two handlers:
+// - read user
+// - return reset password token
+// It has two because the expected route for reading a user overlaps with an already existing one
+// Using `GET /webapi/users/:username` invalidates the `GET /webapi/users/password/token/:token` route
+// An alternative would be using the resource's singular name `GET /webapi/user/:username` but it invalidates the `GET /webapi/user/status` route
+// So, instead we'll use `GET /webapi/users/*wildcard`, parse the path/params and call the appropriate handler
+
+//h.GET("/webapi/users/:username", h.WithAuth(h.getUserHandle))
+//h.GET("/webapi/users/password/token/:token", httplib.MakeHandler(h.getResetPasswordTokenHandle))
```

Comment:
If the above is a limitation in the default parser....

Can you add these two comments to the comment on line 320 as that's where people will be looking to see what routes are supported.